.gitignore

```diff
@@ -23,3 +23,6 @@ go.work.sum
 
 # env file
 .env
+
+# Intellji file
+.idea/
```

Comment:
nit: I would make a separate PR for this change, since it is outside the scope of this commit